src/cli.rs

```diff
@@ -28,25 +38,25 @@ pub fn launch_trin(infura_project_id: String) {
                 Err(_) => panic!("Could not serve from existing path '{}'", path),
                 Ok(()) => unix::net::UnixListener::bind(path).unwrap(),
             }
-        },
+        }
         Err(err) => {
             panic!("Could not serve from path '{}': {:?}", path, err);
         }
     };
 
     for stream in listener.incoming() {
         let stream = stream.unwrap();
-        let infura_project_id = infura_project_id.clone();
+        let infura_project_id = trin_config.infura_project_id.clone();
         pool.execute(move || {
             let infura_url = format!("https://mainnet.infura.io:443/v3/{}", infura_project_id);
             let mut rx = stream.try_clone().unwrap();
             let mut tx = stream;
-            serve_client(&mut rx, &mut tx, &infura_url);
+            serve_ipc_client(&mut rx, &mut tx, &infura_url);
         });
     }
 }
 
-fn serve_client(rx: &mut impl Read, tx: &mut impl Write, infura_url: &String) {
+fn serve_ipc_client(rx: &mut impl Read, tx: &mut impl Write, infura_url: &String) {
```

Comment:
This, `serve_http_client`, and `launch_*_client` all probably make more sense in a new `src/jsonrpc.rs` file, but let's save that for the next PR. (`TrinConfig` though, makes a lot of sense in `cli.rs`, to me)